notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>This was unexpected to me! (a recovering cosmologist)</p><p> </p><p>This 'expectation' should be talked about in the intro, briefly, when you talk about the the end result you're aiming for. Ideally, with links to more information about stellar spectral classes.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>